rust/kcl-lib/src/std/extrude.rs

```diff
@@ -158,6 +159,7 @@ pub(crate) async fn do_post_extrude<'a>(
     sketch: &Sketch,
     solid_id: ArtifactId,
     length: f64,
+    sectional: bool,
```

Comment:
I don't like this addition at all but I couldn't figure out a way to make my filtering below for sectional sweeps to not make some 19 tests fail. Very open to suggestions.